docs/event-bus/10-03-troubleshooting-knative-eventing-upgrade.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Troubleshooting Knative Eventing Upgrade
+type: Troubleshooting
+---
+
+During the upgrade process, there might be certain sporadic instances where the `kyma-installer` gets stuck at the upgrade process of `knative-eventing`. This can happen due to the reason that the upgrade process was unable to clear existing knative subscription which are created internally by Kyma's `event-bus-subscription-controller`. 
```

Comment:
```suggestion
During the upgrade process, there might be certain sporadic instances where the `kyma-installer` gets stuck at the upgrade process of `knative-eventing`. This can happen due to the reason that the upgrade process was unable to clear existing knative subscriptions which are created internally by Kyma's `event-bus-subscription-controller`. 
```